src/ray/gcs/gcs_client/service_based_gcs_client.cc

```diff
@@ -191,11 +193,26 @@ void ServiceBasedGcsClient::ReconnectGcsServer() {
   int index = 0;
   for (; index < RayConfig::instance().ping_gcs_rpc_server_max_retries(); ++index) {
     if (get_server_address_func_(&address)) {
-      RAY_LOG(DEBUG) << "Attemptting to reconnect to GCS server: " << address.first << ":"
-                     << address.second;
+      // After GCS is restarted, the gcs client will reestablish the connection. At
+      // present, every failed RPC request will trigger `ReconnectGcsServer`. In order to
+      // avoid repeated connections in a short period of time, we add a protection
+      // mechanism: if the address does not change, the connection can be made at most
```

Comment:
Why we don't have this policy when the address is changed?

Reply:
If the address is changed, it means GCS is restarted, so we need to reconnect.

Reply:
Ah, I see. So we throttle the reconnection request when it doesn't restart. Can you add a comment like `address does not change (meaning gcs server doesn't restart)` ?

Reply:
Fix it, thanks.